src/appleseed/renderer/kernel/shading/closures.cpp

```diff
@@ -589,13 +585,8 @@ namespace
             };
 
             shading_system.register_closure(name(), id(), params, &prepare_closure, nullptr);
-
             g_closure_convert_funs[id()] = &convert_closure;
-
             g_closure_get_modes_funs[id()] = &modes;
```

Comment:
Do we still need `g_closure_get_modes_funs`?

Reply:
Yes, we use it to pick closures and when computing pdfs